.ci/scripts/coverity.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash -eE
+set -o pipefail
+trap ERR
+
+#
+# This script runs static code analysis using Coverity.
+# The script needs to be run from the main directory.
+#
+export UCC_PASSWORD=$UCC_PASSWORD
+export UCC_USERNAME=$UCC_USERNAME
+echo "THE USER NAME IS $UCC_USERNAME"
+topdir=$(git rev-parse --show-toplevel)
+cd "$topdir" || exit 1
+./autogen.sh
+./configure
+COVERITY_MODULE_NAME="tools/cov-2021.12"
+# module load tools/cov-2021.12
+nproc=$(grep processor /proc/cpuinfo | wc -l)
+make_opt="-j$(($nproc / 2 + 1))"
```

Comment:
'$/${} is unnecessary on arithmetic variables.shellcheck [SC2004](https://www.shellcheck.net/wiki/SC2004)

Reply:
done